BasicTest/Program.cs

```diff
@@ -35,12 +35,12 @@ static void MassiveBulkOpsAsync(int AsyncOpsQty, bool preserveOrder, bool withCo
                 var conn = muxer.GetDatabase();
                 muxer.Wait(conn.PingAsync());
 
-#if DNXCORE50
+#if CORE_CLR
                 int number = 0;
 #endif
                 Action<Task> nonTrivial = delegate
                 {
-#if !DNXCORE50
+#if !CORE_CLR
```

Comment:
Why not just use the loop and `SpinWait.SpinOnce()`? I think it's netstd1.3, so should be available everywhere.